nextcloudappstore/api/v1/release/info.xsd

```diff
@@ -15,7 +15,7 @@
                 <xs:element name="version" type="semver"
                             minOccurs="1" maxOccurs="1"/>
                 <xs:element name="licence" type="licence" minOccurs="1"
-                            maxOccurs="1"/>
+                            maxOccurs="unbounded"/>
```

Comment:
that's what @BernhardPosselt  told me... on IRC... He even said 'looks good' about the PR so it MUST BE PERFECT ;-)